2-scrape-metadata/index.js

```diff
@@ -27,15 +28,15 @@ var pageData = {
 };
 
 converter.fromFile( CSV_FILENAME, function( err, result ) {
-    var eisIdArray = result.map( function( obj ) {
-	return obj.eis_id;
+  var eisIdArray = result.map( function( obj ) {
+    return obj.eis_id;
   } );
-
-    eisIdArray.forEach( function( id ) {
-	request( 'https://cdxnodengn.epa.gov/cdx-enepa-II/public/action/eis/details?eisId=' + id, function( err, res, html ) {
+  Promise.map( eisIdArray, function( id ) {
```

Comment:
yes i agree, i was just going to comment on that. The scraper stops for me after 2k downloads. i would suggest same thing getting it in batches.